lib/src/components/NxList/NxList.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+import { NxListProps, nxListPropTypes, NxListTitleProps } from './types';
+import { splitOutFirst } from '../../util/childUtil';
+import NxListText from './NxListText';
+import NxListSubtext from './NxListSubtext';
+import NxListActions from './NxListActions';
+import NxListButton from './NxListButton';
+import NxListLink from './NxListLink';
+import NxListItem from './NxListItem';
+import NxListError from './NxListError';
+import NxListLoading from './NxListLoading';
+import NxListEmpty from './NxListEmpty';
+import NxListDescriptionTerm from './NxListDescriptionTerm';
+import NxListDescription from './NxListDescription';
+
+const NxListTitle = (props: NxListTitleProps) => {
+  const { children } = props;
+  return (
+    <h3 className="nx-h3">
+      {children}
+    </h3>
+  );
+};
+
+const NxList = (props: NxListProps) => {
```

Comment:
I feel like this is a component that would be worth accepting `ref`s on

Reply:
> I feel like this is a component that would be worth accepting `ref`s on

Support for `ref`s have been added to the `ul` on `NxList`
f3e80d8e16cfeb5df9a1ff3d4576c42c09fe7420

Reply:
> Support for refs have been added to the ul on NxList
f3e80d8

No need for that complexity, it's just `ref={ref}`

Reply:
> No need for that complexity, it's just ref={ref}

Oh I see, it's because you have your own ref too.  For that we use `useMergedRef`.  See `NxTableRow`

Reply:
> > No need for that complexity, it's just ref={ref}
> 
> Oh I see, it's because you have your own ref too. For that we use `useMergedRef`. See `NxTableRow`

Used `mergedRef`
23d7018a62994de918c278396ab479ea24af5ac7